debian/NOTES.md

```diff
@@ -0,0 +1,112 @@
+# NOTES
+
+used ubuntu 20.04 system
+
+## Pre Requisites
+
+```shell
+$ rustup upgrade
+$ sudo apt-get update && sudo apt-get install -y cmake g++ libprotobuf-dev protobuf-compiler
+$ cargo install cargo-deb
+```
+
+## Optinal test build SurrealDB
+
+```shell
+$ make build
+```
+
+## Make Debian
+
+```shell
+$ make debian
+# or 
+$ cargo deb
+# outcome
+...
+info: Generating maintainer script postrm
+info: compressed/original ratio 6274596/20791808 (30%)
+target/debian/surreal_1.0.0~beta.5_amd64.deb
+```
+
+## Install local deb
+
+```shell
+$ sudo dpkg -i target/debian/surreal_1.0.0~beta.5_amd64.deb 
+```
+
+## Some Service 
+
+```shell
+$ sudo service surreal status
+# outcome
+● surreal.service - SurrealDB Service
+     Loaded: loaded (/lib/systemd/system/surreal.service; enabled; vendor preset: enabled)
+     Active: active (running) since Thu 2022-08-11 23:34:35 UTC; 5min ago
+   Main PID: 23177 (surreal)
+      Tasks: 5 (limit: 4605)
+     Memory: 3.2M
+     CGroup: /system.slice/surreal.service
+             └─23177 /usr/share/surreal/surreal start --log trace --user root --pass root
+# other commands
+$ sudo service surreal start
+$ sudo service surreal stop
+$ sudo service surreal enable
+$ sudo service surreal disable
+```
+
+## Test Service
+
+```shell
+$ curl -k -L -s POST \
+  --header "Content-Type: application/json" \
+  --header 'NS: test' \
+  --header 'DB: test' \
+  --user "root:root" \
+  --data "INFO FOR DB" \
+  http://localhost:8000/sql
+# outcome
+[
+  {
+    "time": "75.653µs",
+    "status": "OK",
+    "result": {
+      "dl": {},
+      "dt": {},
+      "sc": {},
+      "tb": {
+        "account": "DEFINE TABLE account SCHEMALESS PERMISSIONS NONE"
+      }
+    }
+  }
+]
+```
+
+## See logs with JournalCtl
+
+```shell
+$ sudo journalctl -f -u surreal
+# or with log
+$ tail -f /var/log/surrealdb.log
```

Comment:
This doesn't do anything at the moment correct? SurrealDB doesn't yet write to a logfile. Or is this done by systemd?

Reply:
Without a log file this is what gives the way to see the logs, yes it works and do the option to see the logs, is the way to work without log filez but best way is having a log file, is what I use in my projects

Reply:
So let's keep `sudo journalctl -f -u surreal` as this works already.

But shall we remove the `tail -f /var/log/surrealdb.log` line, and add a new separate Github Issue for writing all output to a logfile?